app/models/edo_oracle/financial_aid/queries.rb

```diff
@@ -576,6 +576,14 @@ def self.get_awards_by_term_by_type(person_id, aid_year, award_type)
         SQL
       end
 
+      def self.get_financial_resources_links()
+        result = safe_query <<-SQL
```

Comment:
You don't need to assign this to a variable, Ruby will return it

Reply:
I dunno? I'm just following the same pattern used on all the other defs in the file?